next.config.js

```diff
@@ -14,6 +15,15 @@ const nextConfig = {
   env: {
     DEPLOY_PRIME_URL: process.env.DEPLOY_PRIME_URL,
   },
+  async redirects() {
+    return [
+      {
+        source: '/home',
+        destination: '/',
+        permanent: true,
+      },
+    ];
+  },
```

Comment:
In order to avoid the 500 error on the index page RSC fetch `/.rsc?pvg24` I needed to put the homepage somewhere it could be prefetched (at its slug). This redirect will allow the home link of `/home` to be prefetched but redirect to the correct URL.